--- loader.mjs	2022-10-09 18:01:21.417927940 -0700
+++ loader.mjs._new	2022-10-09 17:56:24.248986281 -0700
@@ -3,12 +3,14 @@
     switch (specifier) {
         case "websocket":
             return {
+                shortCircuit: true,
                 url: new URL("./websocket.js", import.meta.url).href,
                 format: "commonjs"
             }
 
         case "http":
             return {
+                shortCircuit: true,
                 url: new URL("./http.js", import.meta.url).href,
                 format: "commonjs"
             }
